### PR TITLE
[4.4]Added an api test for com_menus administrator.

### PR DESCRIPTION
**Pull request by rajputanuj31**
Added an api test for com_menus administrator.

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 6cf38fdfb23983b109dc89652063fe9097842ec1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42665">issues.joomla.org/tracker/joomla-cms/42665</a>.</sub>

File: tests/System/integration/api/com_menus/Administrator.cy.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  it('can deliver a single administrator menu', () => {
+    cy.db_createMenuType({ title: 'automated test administrator menu', client_id: 1 })
+      .then((id) => cy.api_get(`/menus/administrator/${id}`))
+      .then((response) => cy.wrap(response).its('body').its('data').its('attributes')
+        .its('title')
+        .should('include', 'automated test administrator menu'));
+  });
+
+  it('can create a administrator menu', () => {
```
Comment (brianteeman) [inline tests/System/integration/api/com_menus/Administrator.cy.js]:
```suggestion
  it('can create an administrator menu', () => {
```

File: tests/System/integration/api/com_menus/Administrator.cy.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      description: 'The menu for the administrator',
+      menutype: 'menu',
+      title: 'automated test administrator menu',
+    })
+      .then((response) => cy.wrap(response).its('body').its('data').its('attributes')
+        .its('title')
+        .should('include', 'automated test administrator menu'));
+  });
+
+  it('can update a administrator menu', () => {
```
Comment (brianteeman) [inline tests/System/integration/api/com_menus/Administrator.cy.js]:
```suggestion
  it('can update an administrator menu', () => {
```

File: tests/System/integration/api/com_menus/Administrator.cy.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    cy.db_createMenuType({ title: 'automated test administrator menu', client_id: 1 })
+      .then((id) => cy.api_patch(`/menus/administrator/${id}`, { title: 'updated automated test administrator menu' }))
+      .then((response) => cy.wrap(response).its('body').its('data').its('attributes')
+        .its('title')
+        .should('include', 'updated automated test administrator menu'));
+  });
+
+  it('can delete a administrator menu', () => {
```
Comment (brianteeman) [inline tests/System/integration/api/com_menus/Administrator.cy.js]:
```suggestion
  it('can delete an administrator menu', () => {
```

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**